src/main/java/com/ironz/unsafe/UnsafeAndroid.java

```diff
@@ -126,6 +127,29 @@ public boolean compareAndSwapObject(Object obj, long offset, Object expectedValu
     }
 
     /**
+     * Gets an <code>boolean</code> field from the given object.
+     *
+     * @param obj    non-null; object containing the field
+     * @param offset offset to the field within <code>obj</code>
+     * @return the retrieved value
+     */
+    public boolean getBoolean(Object obj, long offset) {
+        return unsafe.getBoolean(obj, offset);
+    }
+
+    /**
+     * Gets an <code>boolean</code> field from the given object,
+     * using <code>volatile</code> semantics.
+     *
+     * @param obj    non-null; object containing the field
+     * @param offset offset to the field within <code>obj</code>
+     * @return the retrieved value
+     */
+    public boolean getBooleanVolatile(Object obj, long offset) {
```

Comment:
Can't find this method in last sources https://android.googlesource.com/platform/libcore/+/android-o-preview-1/ojluni/src/main/java/sun/misc/Unsafe.java